src/Main/PasswordVerifier.java

```diff
@@ -0,0 +1,97 @@
+package Main;
+
+public class PasswordVerifier {
+
+
+    public boolean Verify(String password) throws Exception {
+        boolean validpass = false;
+        boolean  haslowercase = false;
+        int passwordstrength = 0;
+        try {
+            if (passwordnotnull(password)) {
+                passwordstrength = passwordstrength + 1;
+            }
+            if (passwordhasLowercase(password)) {
+                haslowercase = true;
+                passwordstrength = passwordstrength + 1;
+            }
+            if (passwordHasNumber(password)) {
+                passwordstrength = passwordstrength + 1;
+            }
+            if (passwordminLength(password)) {
+                passwordstrength = passwordstrength + 1;
+            }
+            if (passwordhasUppercase(password)) {
+                passwordstrength = passwordstrength + 1;
+            }
```

Comment:
Your decision to have a separate method for each check makes perfect sense given the requirements of the problem. If you had more methods, complex methods, or were working in a larger system you might consider pulling each method out into its own class. Then you could possibly rewrite this to something like:
```java
arrayOfChecks = [];
for (int i = 0; i <= arrayOfChecks.length(); i++) {
    arrayOfChecks[i].verify(password);
    incrementPasswordStrength()
}
```

This might make it easier to extend since when you create a new check you just need to add it to an array.

Both solutions have trade offs. Your solution is likely easier to understand and build for a small problem like this but in a larger application this solution may be more maintainable.

In interviews it is sometimes more important to understand the trade offs of your solution than it is to have the best solution.